src/components/icons/Icons.story.svelte

```diff
@@ -32,6 +37,21 @@
 	<Hst.Variant title="VoteIcon">
 		<Vote {size} />
 	</Hst.Variant>
+	<Hst.Variant title="WeVisIcon">
+		<WeVisIcon {size} />
```

Comment:
I see. Because you didn't specify `viewBox` here, it used the default value from `BaseSvg` which is 16x16 making the icon cropped in the story. Moving `viewBox` into each icon component as mentioned above should fix this issue.

Reply:
I've fixed icon issue. Both in `Footer` and `Icon` are saved.

![image](https://github.com/wevisdemo/parliament-watch/assets/14361087/fa6c02b8-7e4c-486d-a811-ec68bc9fb136)

![image](https://github.com/wevisdemo/parliament-watch/assets/14361087/aca4571e-803d-404c-9465-f1560e1004f9)